server/channels/app/user.go

```diff
@@ -960,11 +966,15 @@ func (a *App) userDeactivated(c request.CTX, userID string) *model.AppError {
 	// bots the user owns. Only notify once, when the user is the owner, not the
 	// owners bots
 	if !user.IsBot {
-		a.notifySysadminsBotOwnerDeactivated(c, userID)
+		if appErr := a.notifySysadminsBotOwnerDeactivated(c, userID); appErr != nil {
+			c.Logger().Warn("Error while notifying system admin of bot owner deactivation", mlog.Err(appErr))
+		}
 	}
 
 	if *a.Config().ServiceSettings.DisableBotsWhenOwnerIsDeactivated {
-		a.disableUserBots(c, userID)
+		if appErr := a.disableUserBots(c, userID); appErr != nil {
+			c.Logger().Warn("Error while disabling user bots", mlog.Err(appErr))
```

Comment:
The log message in L970 is great! It makes it clear that this is happening because the user got deactivated. Can we make the also here here?